x-pack/test/functional/apps/infra/home_page.ts

```diff
@@ -305,7 +312,7 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
             { metric: 'memoryUsage', value: '20.0%' },
           ].forEach(({ metric, value }) => {
             it(`${metric} tile should show ${value}`, async () => {
-              await retry.tryForTime(3 * 1000, async () => {
+              await retry.tryForTime(5000, async () => {
```

Comment:
This test failed on my local, I left it unskipped to see what it does in CI
The failure is that the KPI doesn't show, data is correctly created, checking on the local server I can see the KPI showing correct data but not during the run of the test
![image](https://github.com/user-attachments/assets/0480c816-0f5b-4de3-8bae-8201d4ef4d5d)

Reply:
It also fails on CI, I need to debug more as when opening my local test environment I can see the KPI correctly, so might be something with the test itself

Reply:
Fixed here https://github.com/elastic/kibana/pull/190121/commits/13668dfa069ec4170235230276008860df6e5aaa